user-guide/Standard_Apps/PA/Advanced_Configuration/PA_Integrating_Gateways.md

```diff
@@ -35,6 +35,69 @@ The *PA GW RoutingRules* parameter of a profile instance can contain a set of or
 
 A rule contains one or multiple conditions comparing the field value of the process DOM instance or user DOM instance to fixed values.
 
+### General Build
+
+```json
+{
+    "Evaluation": "First Matching",
+    "Rules": [{
+                "SequenceId": 1,
+                "Description": "Generate a token on each outgoing interface",
+                "OutgoingInterfaceIds": [],
+                "DoTagTokens": true,
+                "Conditions": [{
+                    "IncomingGatewayKeyName": "*",
+                    "Value": "*",
+                    "Operation": "any",
+                    "IncomingInterfaceId": -1
+                },
+                "DomField": {
+                    "FieldDescriptorId": "9a15783b-ccc0-4d8f-9574-0870c55f4a73",
+                    "SectionDefinitionId": "5d6c39ed-f616-4ad2-b311-28e39d52505a"
+                 }
+            ]
+        }
+    ]
+}
+```
+
+### Routing Rule Json Object
+
+#### RoutingRule Object
+
+| Name | Type | Value | Information |
+| --- | --- | --- | --- |
+| Evaluation | string (discrete) | First Matching <br>or All Matching | First Matching: The evaluation will stop when the first rule has a match. <br>All Matching: All rules will be evaluated. Only triggered if all of the conditions match. |
+| Rules | [Rule](#rule-object)[] | [Rule](#rule-object) | |
+
+#### Rule Object
+
+| Name | Type | Value | Information |
+| --- | --- | --- | --- |
+| SequenceId | int |  | Needs to be unique. |
+| Description | string |  | Description for developers. |
+| OutgoingInterfaceIds | int[] | 11, 12, 13, 14, 15 | The list of all the output interface IDs that this rule has to generate a token for. These represent the outgoing interfaces that need to be triggered when this rule matches. |
+| DoTagTokens | bool | true or false | |
```

Comment:
Used on a split gateway to tag all tokens generated from the same incoming token with the same unique identifier. On the associated merge gateway, only tokens with same tag will be considered together to evaluate routing rules.